docs/releases.md

```diff
@@ -0,0 +1,65 @@
+# Release process
+
+The release process in broken up into 3 steps.
+
+1. Increment the version and add a new tag
+2. Modify the changelog
+3. Merge the release branch and create a "github release"
+
+## 1. Increment the version and add a new tag
+
+Run
+
+```shell
+$ yarn version
+```
+
+See [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200).
+
+- MAJOR version when you make incompatible API changes,
+- MINOR version when you add functionality in a backwards compatible manner, and
+- PATCH version when you make backwards compatible bug fixes.
+
+In most cases increment the MINOR version number only. The only change should be to package.json, and there should be a new tag on the commit prefixed with a "v".
+
+## 2. Modify the changelog
+
+Frontend-share is a consumed package, so it is important to emphasize changes that could affect anything that imports it. For this reason, the `CHANGELOG.md` file is used to highlight such changes. `CHANGELOG.md` is based our changelog off [keepachangelog](https://keepachangelog.com/en/1.0.0/_).
```

Comment:
The "Keep a changelog" site explains why a changelog is important, so I think you can omit the first sentence here and revise the second sentence to something like "CHANGELOG.md highlights changes that are relevant for consumers of the package". The last sentence here isn't grammatical. I suggest "Our changelog follows [Keep a Changelog](...)".